tests/framework/ngf.go

```diff
@@ -188,6 +189,16 @@ func UninstallNGF(cfg InstallationConfig, k8sClient client.Client) ([]byte, erro
 	return nil, nil
 }
 
+func setTelemetryArgs(cfg InstallationConfig) []string {
+	var args []string
+
+	args = append(args, formatValueSet("nginxGateway.productTelemetry.enable", "false")...)
+	if cfg.Telemetry {
+		args = append(args, formatValueSet("nginxGateway.productTelemetry.enable", "true")...)
+	}
```

Comment:
This is going to append the value twice if telemetry is enabled (once for false and once for true). Put the `false` condition in an `else` case.

Reply:
But telemetry can either be enabled or disabled. So only one of these conditions can ever be true. 

I believe we spin up new helm installations for every test suite.

So telemetry would be true for longevity but all other cases it would be false.

Reply:
If telemetry is enabled, then this code would have appended both `nginxGateway.productTelemetry.enable=false` and `nginxGateway.productTelemetry.enable=true`